.github/workflows/coverage.yml

```diff
@@ -13,17 +13,18 @@ jobs:
     runs-on: ubuntu-latest
     name: Coverage
     steps:
-      - name: Checkout Code
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v3
```

Comment:
i think we are on actions/checkout@v4 now, maybe checkout the latest public-library-template repo?